content/zh/docs/kitex/Tutorials/advanced-feature/multi_service.md

```diff
@@ -5,16 +5,29 @@ weight: 10
 keywords: ["Kitex", "多 Service", "单 Server 多 Service", "gRPC", "thrift", "protobuf"]
 description: Kitex 支持在一个 Server 上注册多个 Service 。
 ---
+## 介绍
+从 Kitex v0.8.0 开始，支持在单个 Server 上注册多个 Service 。
 
 目前，该功能适用于：
-- gRPC 传输协议
-- Kitex Thrift 和 Protobuf(non-streaming)
+- gRPC 传输协议，包括基于 HTTP2 的 Thrift Streaming (也基于 gRPC ) (>= v0.8.0)
+- Kitex Thrift 和 Protobuf (non-streaming) (>= v0.9.0)
 
 
 ## 使用方法
 
-### 准备工作
-请使用 Kitex 命令工具为每个 Service 生成代码。更多详情，请参考[代码生成工具](/docs/kitex/tutorials/code-gen/code_generation/).
+### 客户端用户
+
+如果您是客户端用户并使用多业务功能，请按照以下说明操作：
+
+1. 将客户端升级到 Kitex 版本 >= v0.9.0
+2. 使用 TTHeader 作为传输协议`client.WithTransportProtocol(transport.TTHeader)`
+3. 在客户端添加以下选项：`client.WithMetaHandler(transmeta.ClientTTHeaderHandler())`
+
+### 服务器端用户
+#### 准备工作
+请使用 Kitex 命令工具 (>= v0.9.0) 为每个 Service 生成代码。更多详情，请参考[代码生成工具](/docs/kitex/tutorials/code-gen/code_generation/)。
```

Comment:
Please replace the link with the chinese doc link: /zh/docs/kitex/tutorials/code-gen/code_generation/

Reply:
done